src/components/WorkshopPage/Workshop.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import ReactPlayer from 'react-player/youtube';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+const workshops = [
+	{
+		type: 'Web Development',
+		elements: [
+			{
+				title: 'Web Dev #1: HTML CSS JS',
+				author: 'Jakob Reinwald',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Web Dev #2: React',
+				author: 'Nareh Agazaryan, Eric Yang',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to Something Else',
+				author: 'Rick Astley',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	},
+	{
+		type: 'Mobile Development',
+		elements: [
+			{
+				title: 'Mobile Dev with React Native',
+				author: 'Einar Balan',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Android Studio',
+				author: 'Nareh',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	}
+];
+
+function Workshop() {
```

Comment:
nit: a more descriptive name would be quite helpful here. The component returns the whole workshop page, not a singular workshop, so something like `WorkshopPage` would probably be better.